.github/workflows/web-api.yml

```diff
@@ -86,3 +87,31 @@ jobs:
         uses: codecov/codecov-action@v1
         with:
           files: coverage.xml
+
```

Comment:
same here.. we should keep the triggers to run the workflow only on specific file changes